tests/runtime/model_management/test_model_manager.py

```diff
@@ -1054,3 +1055,97 @@ def test_lazy_load_handles_temporary_errors():
                 assert manager._lazy_sync_timer is None
                 model = manager.retrieve_model(model_name)
                 assert model
+
+
+def test_lazy_load_true_local_models_dir_valid():
+    """When lazy_load_local_models is True and local_models_dir exists.
+    Check that the local_models_dir is pointing to the correct location
+    """
+
+    with TemporaryDirectory() as cache_dir:
+
+        ModelManager._ModelManager__instance = None
+        with temp_config(
+            {
+                "runtime": {
+                    "local_models_dir": cache_dir,
+                    "lazy_load_local_models": True,
+                    "training": {"save_with_id": False, "output_dir": cache_dir},
```

Comment:
why do we need `training`?

Reply:
Good question. I think we dont, so will take it out
Was trying to mimic the https://github.com/caikit/caikit/issues/545 env specifically